homeassistant/components/xiaomi_aqara/binary_sensor.py

```diff
@@ -181,6 +182,12 @@ def extra_state_attributes(self):
         attrs.update(super().extra_state_attributes)
         return attrs
 
+    async def async_added_to_hass(self) -> None:
+        """Handle entity which will be added."""
+        await super().async_added_to_hass()
+        self._state = False
```

Comment:
Why do we set it to False instead of restoring state?

Reply:
For Gas, Smoke, WaterLeak and Motion sensors it made more sense to me to set the state to False at the start of HomeAssistant.
If for instance Motion is detected just before HomeAssistant restarts/updates the last state will be motion_detected.
In the time HomeAssistant takes to restart the motion_detected will probably already be old.
More over if it is not, the motion sensor will send motion_detected messages at a intervall of 1 minute while motion is continuesly detected, so the sensor will be in sync again within 1 minute after restart.

However if motion is not anymore detected when HomeAssistant started, the state will get stuck at motion_detected since the sensor will not send any messages anymore. Only once a new motion is detected and then after 2 minute of no motion the state will go back to not_detected.
So you run the risk of having the sensor stuck at motion_detected untill another motion is detected.

Reply:
Ok.